plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -368,6 +361,27 @@ public static function verify_base_tables( $modify_notice = true, $execute = fal
 		return $missing_tables;
 	}
 
+	/**
+	 * Verify if the table(s) already exists.
+	 *
+	 * @param string $schema Schema definition to check against.
+	 *
+	 * return array List of missing tables.
+	 */
+	public static function verify_schema( $schema ) {
```

Comment:
If we are going to add this as a public method I think we need to come up with a better name, `verify_schema` is too generic and the same for the argument. Maybe something like `verify_database_tables_exist( $tables_creation_sql )`?

Or even better, since this method is public solely because we need to invoke it from another class that is `Internal`, why don't we move this method to an internal class too (maybe something like `DbUtils`), then invoke it from inside `verify_database_tables` too? That would require a bit of setup with the DI container though, but I can help with that.